jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/impl/model/EmailNotificationImpl.java

```diff
@@ -32,8 +32,8 @@
 import org.kie.internal.task.api.model.Language;
 import org.kie.internal.task.api.model.NotificationType;
 
-
-@Entity
+@Entity(name = "Notification") // need to use `name = "Notification"` to make the DDL scripts work with both Hibernate 4 and Hibernate 5
+// see https://issues.jboss.org/browse/RHBPMS-4244 for more info
```

Comment:
I don't have reasonably good explanation :) Sometimes when I do something non-obvious (which in this case is naming the entity by the parent class name) I tend to also add a JIRA link to make it more visible. But I may just be overthinking this...it's true that the JIRA number will always be part of the git history.